Adding docs for TELCODOCS-1985: Day-2 upgrades for telco core clusters (#83448)
openshift/openshift-docs: modules/telco-upgrade-acknowledging-the-y-stream-release-upgrade.adoc

// Module included in the following assemblies:
//
// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc

> 

Reviewer: Missing the procedure designation

[id="telco-upgrade-acknowledging-the-y-stream-release-upgrade_{context}"]
= Acknowledging the y-stream release upgrade

> When moving between y-stream releases you need to run a patch command to explicitly acknowledge the upgrade.

Reviewer: ```suggestion
When moving between y-stream releases, you must run a patch command to explicitly acknowledge the upgrade.
```

> In the output of the `oc adm upgrade` command a URL is provided that shows the specific command to run.

Reviewer: ```suggestion
In the output of the `oc adm upgrade` command, a URL is provided that shows the specific command to run.
```

> .Procedure

Reviewer: Same comment about verifying the removed Kubernetes APIs aren't in use first

> . Acknowledge the y-stream release upgrade by patching the `openshift-config` resource.

Reviewer: `openshift-config` is the namespace, not the resource. The resource the `admin-acks` config map.

```suggestion
. Acknowledge the y-stream release upgrade by patching the `admin-acks` config map in the `openshift-config` namespace.
```
